tests/h/schemas/forms/accounts/reset_password_test.py

```diff
@@ -1,86 +1,128 @@
+from datetime import datetime, timedelta
+from unittest.mock import create_autospec
+
 import colander
 import pytest
-from itsdangerous import BadData, SignatureExpired
+import pytz
+from itsdangerous import BadData, SignatureExpired, URLSafeTimedSerializer
 
 from h.schemas.forms.accounts import ResetPasswordSchema
 
-pytestmark = pytest.mark.usefixtures("pyramid_config")
-
 
-@pytest.mark.usefixtures("user_model")
```

Comment:
This is a locally defined fixture, so it now has `autouse=True`. Also we now mock the `model` import not the source object.